clap_mangen/src/render.rs

```diff
@@ -28,8 +28,14 @@ pub(crate) fn description(roff: &mut Roff, cmd: &clap::Command) {
     }
 }
 
+fn get_command_name(cmd: &clap::Command) -> &str {
+    cmd.get_bin_name()
+        .or_else(|| cmd.get_display_name())
+        .unwrap_or_else(|| cmd.get_name())
+}
```

Comment:
The main risk for this is if the user parsed with the `Command` before calling this, leading it to have a `.exe` in the root bin name which can cause problems with this approach